src/main.rs

```diff
@@ -102,7 +102,10 @@ async fn main() {
     if let Some(mut opts) = launch_opts {
         // Warn about missing launch options for TF2
         let missing = opts.check_missing_args();
-        if args.rewrite_launch_options {
+        if let Err(err) = missing {
+            tracing::error!("Failed to check for missing launch arguments: {}", err);
+            panic!("Missing required launch options in TF2 for MAC to function. Aborting...");
```

Comment:
As of now, if there are no user configured launch options the check_missing_args function will raise an error but will proceed through without properly checking the args. Should this be handled in the check_missing_args or in main? I still think it should panic because it should act similarly to if we are missing args.

Reply:
> because it should act similarly to if we are missing args
The current behaviour for that is to panic unless we are ignoring them, like I suggest we do here.

Additionally, missing launch options does not inhibit all features of the program (only the TF2 interactions) so there may still be a use-case without using those features, and perhaps there is some other scenario we don't know about that will cause this to fail, in which case we want to be able to bypass this check manually.

Reply:
Ah I see, misunderstanding on my part. Made changes that should find the err and check ignore_launch_options before a panic again.